frontend/api/baseApi.ts

```diff
@@ -8,28 +8,28 @@ export class BaseApi {
 		this.client = createAxiosInstance();
 	}
 
-	private get(url: string, params?: Record<string, any>) {
-		return this.client.get(url, {
+	private get<T>(url: string, params?: Record<string, any>) {
+		return this.client.get<T>(url, {
 			params,
 		});
 	}
 
-	private post(url: string, body?: Record<string, any>) {
-		return this.client.post(url, {
+	private post<T>(url: string, body?: Record<string, any>) {
```

Comment:
this gets nested to `req.body.body`, can pass body directly as second arg:

```
this.client.post<T>(url, body);
```

Reply:
yes picked up on this yesterday. i think the fix is on another branch i'm working on. will fix it here too